### DISCUSSION TITLE
웹 반응형 너비 관련 discussion입니다.

### DISCUSSION THREAD

**Discussion by dldnjswns31** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#13] [Trouble Shooting] [answered]
모바일 반응형을 360px~450px까지 생각하랬는데 너비 390px인 iPhone 12 Pro로 봤을때  화면이 다음과 같이 짤립니다

![image](https://user-images.githubusercontent.com/71388830/199722514-46987b84-7acd-440a-86d9-ebbe75ffe85b.png)

너비 관련 수정 들어가야할것같습니다.

**Comment by jaeyeong815** [comment, marked as answer]
GlobalStyle.js에 있는 .App 속성에 ```  margin-left: auto;  margin-right: auto;``` 추가하여 해결하였습니다.
@yeondooo 연주님 감사합니다!
